airflow/www/templates/airflow/variable_edit.html

```diff
@@ -0,0 +1,28 @@
+{#
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+#}
+{% extends 'appbuilder/general/model/edit.html' %}
+
+{% block tail %}
+  {{ super() }}
+  <script>
+    let val = document.getElementById('val');
+    val.style.fontFamily = 'monospace';
```

Comment:
Is this the best way to do this? From what I can tell the other option involves essentially duplicating/re-writing the `FormWidget` template.

Reply:
I would rather we did it with a style tag in this page:

```html
<style>
#val { font-family: monospace };
</style>
```

Reply:
Could you add `[AIRFLOW-310]` in to your commit messages and PR title (i.e. `[AIRFLOW-4929][AIRFLOW-310]`)

Reply:
> I would rather we did it with a style tag in this page

Ohh right, of course!

> Could you add [AIRFLOW-310] in to your commit messages and PR title

Sure, though I should mention that 310 lists a couple of features that are not included here (syntax highlighting for example). 4929 is fairly vague, so I figured the font change could be considered to be within the scope of that issue. Should I still add 310 to the title? Alternatively I could add a comment to 310 on Jira noting that this feature has been added in this PR.

Reply:
OK I changed this to use CSS to set the font and added a comment on [AIRFLOW-310](https://issues.apache.org/jira/browse/AIRFLOW-310).